src/components/CourseOutline.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts">
+import { useCourseStore } from '../stores/courses'
+import { storeToRefs } from 'pinia'
+import { onMounted } from 'vue'
+const { currentCourse } = storeToRefs(useCourseStore())
+const { getCurrentCourse } = useCourseStore()
+import { DateTime } from 'luxon'
+
+onMounted(async () => {
+  await getCurrentCourse()
+})
+</script>
+
+<template>
+  <v-container>
+    <v-row>
+      <v-col cols="12">
+        <v-expansion-panels
+          v-for="(module, index) in currentCourse?.chapters"
+          :key="index"
+          class="eleva mt-2"
+        >
+          <v-expansion-panel elevation="0">
+            <v-expansion-panel-title class="py-0 px-1">
+              <v-row>
+                <v-col class="d-flex">
+                  <BaseButton
+                    color="rgba(0, 0, 0, 0.03)"
+                    icon="mdi-plus"
+                    buttonClass="elevation-0 mt-2"
+                    buttonSize="x-small"
+                  >
+                    {{ index + 1 }}
+                  </BaseButton>
+                  <v-card-item class="d-flex justify-center align-items-center">
+                    <p style="font-size: 16px" class="mb-1">{{ module.display_name }}</p>
+                    <span style="font-size: 11px">
+                      {{ DateTime.fromISO(module.start).toFormat('dd LLL') }} -
+                      {{
+                        DateTime.fromISO(currentCourse?.module_end_date[index]).toFormat('dd LLL')
+                      }}
+                      <span>
+                        {{
+                          Math.ceil(
+                            DateTime.fromISO(module.start).diff(
+                              DateTime.fromISO(currentCourse?.module_end_date[index]),
+                              'days'
+                            ).days
+                          )
+                        }}
+                        days
+                      </span>
+                      <span class="ml-2" v-if="module.complete">
+                        <img src="../assets/icons/completed.svg" />
+                        Completed
+                      </span>
+                      <span class="px-2" v-else>
+                        <img src="../assets/icons/ongoing.svg" />
+                      </span>
+                      <span>Ongoing</span>
+                    </span>
+                  </v-card-item>
```

Comment:
@AishwaryShrivastav  this is the lower section where the dates shows in course details page .

 **Doubt in this two things** 
_-- course ongoing and completed icon slightly showing upper side,not able to inline with text.
-- Not added big logic to show the course is ongoing or completed. Just get the complete : true or false from API and showed if course is ongoing or completed_